app/views/grade_entry_forms/_form.html.erb

```diff
@@ -89,3 +89,13 @@
                  data: { disable_with: t('working') } %>
   </p>
 <% end %>
+
+<% unless @grade_entry_form.id.nil? %>
```

Comment:
Oh after you modified this I realized exactly what you were checking; use the `new_record?` method instead to check whether a model has been saved or not.